### PR TITLE
Integrate "Manage sender identity" endpoints

### PR DESCRIPTION
**Pull request by PeterOcansey**
resolves #https://github.com/mailersend/mailersend/issues/2845

### Changelist
- [x] Add list sender identities
- [x] View single sender identity
- [x] Create a sender identity
- [x] Update a sender identity
- [x] Delete a sender identity
- [x] Update README

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -27,7 +27,8 @@
     "psr/http-client-implementation": "^1.0",
     "psr/http-message": "^1.0",
     "tightenco/collect": "^7.0|^8.0",
-    "beberlei/assert": "^3.2"
+    "beberlei/assert": "^3.2",
+    "symfony/symfony": "5.4.16"
```
Comment (JoaoFSCruz) [inline composer.json:31]: Why are you pulling symfony?

File: README.md [hunk 2]
```diff
@@ -1827,6 +1833,103 @@ $mailersend = new MailerSend(['api_key' => 'key']);
 $smsRecipients = $mailersend->smsInbound->delete('sms_inbound_id');
 ```
 
+<a name="sender-identity-routing"></a>
+
+## Sender identity routing
+
+<a name="get-a-list-of-inbound-routes"></a>
```
Comment (JoaoFSCruz) [inline README.md:1840]: This link needs to be fixed

File: README.md [hunk 3]
```diff
@@ -104,6 +104,12 @@ MailerSend PHP SDK
         * [Add an SMS inbound route](#create-sms-inbound)
         * [Update an inbound route](#update-sms-inbound)
         * [Delete an SMS inbound route](#delete-sms-inbound)
+    * [Sender Identity](#sender-identity-routing)
+        * [Get a list of sender identity routes](#get-a-list-of-sender-identity-routes)
+        * [Get a single sender identity route](#get-a-single-sender-identity-route)
+        * [Add a sender_identity route](#add-a-sender-identity-route)
+        * [Update a sender_identity route](#update-a-sender-identity-route)
+        * [Delete a_sender_identity route](#delete-an-sender-identity-route)
```
Comment (JoaoFSCruz) [inline README.md:112]: The text and link need fixing

**Review by JoaoFSCruz — CHANGES_REQUESTED** [review]
It looks good! Just some small tiny bits 👌

File: composer.json (hunk 1 above)
Comment (PeterOcansey) [reply composer.json:31]: There was an issue with code style and test pass on php 8 but fails on php 7.4, and it had to do with an update on symfony. Kindly find full discussions here:

https://themailergroup.slack.com/archives/C02657BK2MU/p1671610967361939

Consideration was to lock down symfony to a specific version that works with php 7.4

**Pull request merged**